releasenotes/notes/fix-fluentd-opensearch-template-14bde9b6d5a723f8.yaml

```diff
@@ -0,0 +1,4 @@
+---
+fixes:
+  - |
+    Fix missing bracket in fluentd opensearch config.
```

Comment:
It could have been useful to mention that it only impacts configurations where caso is enabled.